core/src/main/java/hudson/model/Node.java

```diff
@@ -138,6 +138,7 @@ public boolean isHoldOffLaunchUntilSave() {
     }
 
     /**
+     * In most cases, you should not call this method directly, but {@link Jenkins#updateNode(Node)} instead.
```

Comment:
Perhaps this should be overridden in `Slave` and that overload `@Deprecated` as a reminder? In fact https://github.com/jenkinsci/jenkins/blob/f964e565ee54fda6c006e02cac5b23e9ef43fa62/core/src/main/java/hudson/model/Node.java#L280-L284 looks suspicious.

Reply:
4859c37